src/main/kotlin/no/nav/familie/ba/sak/kjerne/logg/LoggService.kt

```diff
@@ -411,6 +411,24 @@ class LoggService(
         )
     }
 
+    fun opprettSmåbarnstilleggLogg(
+        behandling: Behandling,
+        tittel: String
+    ) {
+        lagre(
+            Logg(
+                behandlingId = behandling.id,
+                type = LoggType.MANUELT_SMÅBARNSTILLEGG_JUSTERING,
+                rolle = SikkerhetContext.hentRolletilgangFraSikkerhetscontext(
+                    rolleConfig,
+                    BehandlerRolle.SAKSBEHANDLER
+                ),
+                tittel = tittel,
+                tekst = ""
+            )
+        )
+    }
```

Comment:
Kan slippe litt indentering ved å ha som expression body
```suggestion
  =  lagre(
          Logg(
              behandlingId = behandling.id,
              type = LoggType.MANUELT_SMÅBARNSTILLEGG_JUSTERING,
              rolle = SikkerhetContext.hentRolletilgangFraSikkerhetscontext(
                  rolleConfig,
                  BehandlerRolle.SAKSBEHANDLER
              ),
              tittel = tittel,
              tekst = ""
          )
      )    
```